src/metadata.js

```diff
@@ -505,6 +505,28 @@ export function getIdentifiersFromUris(projectId, uris) {
 }
 
 /**
+ * Get attribute elements with their labels and uris.
+ *
+ * @param projectId id of the project
+ * @param labelUri uri of the label (attribute display form)
+ * @param mode match mode, only 'EXACT' supported
+ * @param patterns [ STRING ] element labels (in EXACT mode) or patterns to match them
+ */
+export function getElementLabelToUri(projectId, labelUri, mode, patterns) {
```

Comment:
Since only EXACT is currently supported, how about `function getElementLabelToUri(projectId, labelUri, patterns, mode = 'EXACT') {...}`?

Reply:
I do not see a problem to change it :)